pkg/kubectl/cmd/cmd.go

```diff
@@ -65,6 +65,7 @@ Find more information at https://github.com/GoogleCloudPlatform/kubernetes.`,
 	cmds.AddCommand(NewCmdClusterInfo(f, out))
 	cmds.AddCommand(NewCmdApiVersions(f, out))
 	cmds.AddCommand(NewCmdVersion(f, out))
+	cmds.AddCommand(NewCmdValidateCluster(f, out))
```

Comment:
We have `cluster-info`. How about `cluster-validate`? `validate` is just too generic, and we're likely to want to use that for validating user objects.